### ISSUE TITLE
Enable travis runs for pull requests.

### ISSUE THREAD

**Issue by thomasdziedzic** [MartijnSch/optimizely-gem#11]
I opened https://github.com/MartijnSch/optimizely-gem/pull/10 and I expected travis to run automatically. It did not.

**Comment by MartijnSch** [comment]
It would be good to have this, did your PR fix this?

**Comment by thomasdziedzic** [comment]
No, I think you have to setup the travis integration as the project owner.

Check out https://docs.travis-ci.com/user/getting-started/
